ci/c0-azure-concourse-poc-params.yml

```diff
@@ -24,37 +24,43 @@ pcf_ssh_key_priv: |
 # ERT Cert & Key , if the string 'generate' is passed, pipeline will autogen certs
 pcf_ert_ssl_cert: |
 
-pcf_ert_ssl_key: |
+pcf_ert_ssl_key:
 
 ##############################################
 ### IaaS Specific Params
 ##############################################
 # Wipe Arg(s) anything other than wipe bypasses the wipe-env job
 arg_wipe: wipe
 ### azure ###
+azure_access_key:
+azure_account_name:
+azure_buildpacks_container:
+azure_droplets_container:
+azure_packages_container:
```

Comment:
It would be good to document these `*_container` params. What are they for? Is container synonymous with S3 bucket here?